lib/core/src/Cardano/Wallet/DB/Checkpoints/Model.hs

```diff
@@ -8,38 +9,54 @@
 --
 -- Data type that represents a collection of checkpoints.
 -- Each checkpoints is associated with a 'Slot'.
-
-module Cardano.Wallet.DB.Checkpoints
-    ( -- * Checkpoints  
-      Checkpoints
+module Cardano.Wallet.DB.Checkpoints.Model 
+    (
+    -- * Checkpoints
+    Checkpoints
     , checkpoints
     , loadCheckpoints
     , fromGenesis
     , getLatest
     , findNearestPoint
-    
+
     -- * Delta types
     , DeltaCheckpoints (..)
-    ) where
 
-import Prelude
+    -- * Wallet state component
+    , getBlockHeight
+    , getSlot
+    , toWallet
+    , fromWallet
+    , WalletCheckpoint (..)
+    ) where
 
+import Cardano.Wallet.DB.Checkpoints.AddressBook
+    ( AddressBookIso (Discoveries, Prologue), addressIso )
+import qualified Cardano.Wallet.Primitive.Model as W
+import Cardano.Wallet.Primitive.Types
+    ( BlockHeader )
+import qualified Cardano.Wallet.Primitive.Types as W
+import Cardano.Wallet.Primitive.Types.UTxO
+    ( UTxO )
```

Comment:
Hi @paolino 

In our [coding standards document](https://github.com/input-output-hk/adrestia/blob/master/docs/code/Coding-Standards.md), it specifies to [separate imports into three distinct groups](https://github.com/input-output-hk/adrestia/blob/master/docs/code/Coding-Standards.md#stylish-haskell-is-used-to-format-grouped-imports--language-pragmas):

> In order to maximize readability, imports should be grouped into three groups, separated by a blank newline.
>
>   Prelude import
>   Explicit imports
>   Qualified imports

We've used this style throughout the wallet, so that when people look for a given import, they can search for imports in a predictable way.

Sorry to ask, but would you be able to go through this PR and make sure that each of the modules conforms to this rule?

Many thanks!

Reply:
Yes, it slipped through, sorry